recipes/flask-cache/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "flask-cache" %}
+{% set version = "0.13.1" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version}}.tar.gz
+  url: https://pypi.python.org/packages/91/c4/f71095437bd4b691c63f240e72a20c57e2c216085cbc271f79665885d3da/Flask-Cache-0.13.1.tar.gz
```

Comment:
Maybe you could try copying pieces from [this]( https://github.com/conda-forge/staged-recipes/blob/89d8a420415e13b25a26e2cfcd1a123009cc491a/recipes/pyinstaller/meta.yaml ) as a template.